### DISCUSSION TITLE
Whitelist IDs or devices

### DISCUSSION THREAD

**Discussion by rubslopes** [rustdesk/rustdesk#5414] [Feature Request]
RustDesk already allows you to whitelist IPs, but many people have dynamic IPs. It'd be great to be able to whitelist IDs, or devices associated with an account. That would make me sleep better at night when leaving my computer open to connections.

**Comment by Dex321x** [comment]
I would like a feature like this too to add extra security. I have a private server where I manage some devices and I would like to add a whitelist RustDesk ID to the client for allow, what client can connect and which can't.

**Comment by pdrive** [comment]
any plans for this feature?

**Comment by Avrution** [comment]
Still can't believe no response or update on this in over 2 years.  Such a basic feature really should exist - IP whitelisting really limits the chance anyone will actually use the feature.

**Comment by 21pages** [comment]
Available in the latest RustDesk Server Pro release (https://github.com/rustdesk/rustdesk-server-pro/releases/tag/1.8.6) and the RustDesk nightly build (https://github.com/rustdesk/rustdesk/releases/tag/nightly).

**Comment by Tobiiiiu** [comment]
the feature also apears to be in the nightly build for the 1.5.0
